src/main/webapp/scripts/script.js

```diff
@@ -12,17 +12,10 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+
 /**
- * Adds a random greeting to the page.
+ * Function to load the search vendors around bar
  */
-function addRandomGreeting() {
-  const greetings =
-      ['Hello world!', '¡Hola Mundo!', '你好，世界！', 'Bonjour le monde!'];
-
-  // Pick a random greeting.
-  const greeting = greetings[Math.floor(Math.random() * greetings.length)];
-
-  // Add it to the page.
-  const greetingContainer = document.getElementById('greeting-container');
-  greetingContainer.innerText = greeting;
-}
+$(function(){
+  $("#searchBarVendors-placeholder").load("COMMONS/searchBarVendors.html"); 
+});
```

Comment:
Looks good, but what do you think about changing the name of the file to something more descriptive?

Reply:
(I mean the JS file)

Reply:
For convenience in the project, we will have a script.js file for the html pages. In this file we will include more functions, possibly in the future if we split the scripts in different file a better name will be helpful, but for this phase I don't think its necessary